app/views/infuse.html

```diff
@@ -0,0 +1,44 @@
+<div ng-controller="dimInfuseCtrl as infuse" dim-click-anywhere-but-here="app.closeLoadoutPopup()">
+  <h2>Infusion Calculator</h2>
+
+  <div style="margin-left: 25px;">
```

Comment:
maybe we could consider moving out some of these inline styles, that way we could potentially reuse some of them. we might even have some CSS classes that you can even already use here.